### ISSUE TITLE
drag highlighting is funky when it overlaps highlighted text

### ISSUE THREAD

**Issue by daviddurand** [tschicke-brown/FRESSClient#5]
At first I thought that I'd failed in creating the Hypertext tag highlighting, but I just realized that it happens with italics and bold highlight codes as well, so it's not me...

**Comment by tschicke-brown** [comment]
By highlighted text, do you mean formatted (italic/bold) text? If so, I know what the issue is and was working on fixing it before. I can finish it up now if you need it.

**Comment by daviddurand** [comment]
Yes, that's the problem. There's no rush, I suppose, but it would definitely be harder for me to fix.

**Comment by daviddurand** [comment]
bump. It would be great to fix this before the event in May.

**Comment by tschicke-brown** [comment]
I can try to take a look at this in the next couple weeks
